changelog.md

```diff
@@ -2,6 +2,10 @@
 
 Since you are interested in what happens next, in case, you work for a for-profit company that benefits from using the project, please consider supporting it on https://opencollective.com/jss.
 
+### Bug fixes
+
+- [jss] Fix breaking change that was introduced in `10.0.1` ([1252](https://github.com/cssinjs/jss/pull/1252))
+
 ---
```

Comment:
uhh, that thing was supposed to be above the changelog entries, that's why release message was broken, we should validate this somehow or make it more obvious